api/app/db/crud/hfi_calc.py

```diff
@@ -78,3 +78,8 @@ def get_fire_centre_fire_start_ranges(session: Session, fire_centre_id: id) -> C
 def get_fire_start_lookup(session: Session) -> CursorResult:
     """ Get the fire start lookup table """
     return session.query(FireStartLookup)
+
+
+def get_fuel_types_from_db(session: Session) -> CursorResult:
```

Comment:
```suggestion
def get_fuel_types(session: Session) -> CursorResult:
```

I think "from_db" is a bit redundant - it's in app.db.crud.hfi_calc - so I think in that context it's clear.

Reply:
I agree "from_db" is redundant, but it was the quickest solution I could think of. In routers.hfi_calc, the /fuel_types endpoint's method is called "get_fuel_types", and when I'd previously called the crud method "get_fuel_types" too, there was an infinitely recursive loop in routers.hfi_calc where that file's "get_fuel_types" was calling itself.

Do you know how to get around that?

Reply:
For sure. Either do an "from blah import thing as moo" or do an "import blah", and the call it by saying blah.thing.

Reply:
If you're wondering - yes, I am embarassed that I asked that. I don't know why I didn't think of that.